third_party/docfx/templates/devsite/UniversalReference.common.js

```diff
@@ -114,6 +114,11 @@ function handleItem(vm, gitContribute, gitUrlPattern) {
   vm.inheritance = vm.inheritance || null;
   vm.children = vm.children || null;
   vm.status = vm.status || null;
+
+  if (!common.isValidPackageVersion(vm._packageVersion)) {
+    vm._packageVersion = null;
+  }
+

```

Comment:
Marking the version information simply as NULL could result in more confusion on the information missing (and SEO issues?). I would prefer we process the version as-is, rather than doing our own modifications for unintended consequences.

Reply:
Changed to leave version as-is, no modifications.